### PR TITLE
Adding version file

### PR DESCRIPTION
**Pull request by abhidnya13**
Adding this file here to track version number.

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
Is there a reason the wiring up to `comm.go` needs to happen in a separate PR?

**Comment by abhidnya13** [comment]
I tried changing it here but vs code was complaining that there is no version folder to import this value from

**Comment by element-of-surprise** [comment]
@abhidnya13  That is some type of local issue.  Probably will happen at the cmd line too.  

Please figure out what is going on and update the comm package to use this.  Also, go ahead and wire the version up to be 0.1.0 so we don't have to do this later.

**Comment by abhidnya13** [comment]
@jhendrixMSFT , @element-of-surprise Made the changes here.

File: apps/internal/version/version.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package version
```
Comment (element-of-surprise) [inline apps/internal/version/version.go:4]: I hate to say this, but you have to put a // Package version .... here

**Review by element-of-surprise — APPROVED** [review]

**Pull request merged**